eox_hooks/models.py

```diff
@@ -0,0 +1,36 @@
+"""Models module for eox-hooks."""
+import uuid
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+from eox_hooks.constants import Status
+
+
+@python_2_unicode_compatible
+class HookExecutionAudit(models.Model):
+    """Model used to audit the eox-hooks actions."""
+
+    key = models.UUIDField(
+        unique=True,
+        editable=False,
+        default=uuid.uuid4,
+        verbose_name="Public identifier",
+    )
+
+    status = models.PositiveIntegerField(
+        choices=Status.choices(), default=Status.SUCCESS,
+    )
+    executed_at = models.DateTimeField(auto_now_add=True, editable=False)
+    trigger_event = models.CharField(max_length=150)
+    action_name = models.CharField(max_length=150)
+    tenant_domain = models.CharField(max_length=150)
+    action_duration = models.DurationField(null=True, blank=True)
+
```

Comment:
Is it possible to add a text field here where we can write the whole execution log? with timestamps and tracebacks and all that

Reply:
Well, I added the text field. I guess we could extract that info from an exception